api/src/main/resources/workbench.yaml

```diff
@@ -684,6 +710,36 @@ definitions:
         type: integer
         format: int64
         description: Milliseconds since the UNIX epoch.
+      userRoles:
+        type: "array"
+        items:
+           $ref: "#/definitions/UserRole"
+
+  UserRole:
+    type: object
+    required:
+      - user
+      - role
+    properties:
+      user:
```

Comment:
I'd prefer either `user_id` or `email` here, to make it explicit how we're referencing users.

Reply:
Will change.